designs/administrator/4.4/notification-drawer/index.md

```diff
@@ -9,6 +9,11 @@ version: 4.4
 ![](img/notifications-closed.png)
 - When alerts are firing, a special signifier icon appears in the masthead for users with appropriate RBAC permissions. Clicking on the icon opens the notification drawer.
 
+![](img/notifications-icon.png)
+- When there are no notifications, the masthead icon displays only a bell.
+- When there are non-critical notifications (such as updates, lower severity alerts, or other messages) a blue dot appears on the icon along with a number representing the number of items in the drawer,
+- When critical alerts are firing, the icon displays a red (!) icon along with the the total item count.
```

Comment:
In the case when the critical icon is shown, is the number associated with the number of critical notifications only?

Reply:
Yes that is my understanding

Reply:
I take that back! Chris explain this morning in the admin stakeholder meeting that the count will reflect the total.